app/models/__init__.py

```
@@ -0,0 +1 @@
+from .models import *
```

Ruff (0.15.12)

[error] `from .models import *` used; unable to detect undefined names

(F403)